### PR TITLE
[org] add support for transclusion

### PR DESCRIPTION
**Pull request by Kazark**
I am new to `org-transclusion` and somewhat inexperienced at adding new packages and coming up with keybindings, so I'm very open to feedback.

### CODE REVIEW

File: layers/+emacs/org/README.org [hunk 1]
```diff
@@ -551,6 +554,15 @@ To install [[https://github.com/awth13/org-appear][org-appear]], which toggles v
 If you set =org-appear-trigger= to =manual= and your editing style is =vim= or =hybrid=,
 =org-appear= is turned on in insert mode but not in normal mode.
 
+** Transclusion support
+To install [[https://github.com/nobiot/org-transclusion][org-transclusion]], which includes a read-only copy of textual content
```
Comment (lebensterben) [inline layers/+emacs/org/README.org:558]: I suggest
```
..., which allows you to include a view of another file via a file link or ID link, set the ....
```

File: layers/+emacs/org/packages.el [hunk 2]
```diff
@@ -1040,6 +1041,21 @@ Headline^^            Visit entry^^               Filter^^                    Da
                   (add-hook 'evil-insert-state-entry-hook #'org-appear-manual-start nil t)
                   (add-hook 'evil-insert-state-exit-hook #'org-appear-manual-stop nil t))))))
 
+(defun org/init-org-transclusion ()
+  (use-package org-transclusion
+    :defer t
+    :init
```
Comment (lebensterben) [inline layers/+emacs/org/packages.el:1047]: Our convention is that, if `:init` or `:config` of `use-package` has multiple forms, they need to be wrapped in a `progn`.

**Review by lebensterben — APPROVED** [review]
LGTM. Only minor changes are required.

**Comment by Kazark** [comment]
Thanks for the review. I've got a pretty busy next couple days, but I will hope to circle back to this early next week.

**Comment by lebensterben** [comment]
@Kazark 
Just give the push access and I can modify that.

**Comment by Kazark** [comment]
Looks like it's already checked. Thanks.

**Comment by lebensterben** [comment]
Merged. Thanks for you contribution.

**Pull request merged**